account-kit/logging/src/index.ts

```diff
@@ -0,0 +1,13 @@
+import { createClientLogger } from "./client.js";
+import { createServerLogger } from "./server.js";
+import type { EventLogger } from "./types";
+
+export type { EventLogger };
+
+export function createLogger(): EventLogger {
```

Comment:
should we create a static logger object that just get's imported instead of function to create a new logger every time?

Reply:
I actually want to create a static logger **per** package. This will actually take in some `context` object as well that will include like the package name + version and that will be added to each event.

So in each package's `index.ts` I'll have `export const logger = createLogger({ package: "@account-kit/react", version })`

Reply:
will update the context piece once I actually consume this from somewhere so I can get a feel for it. gonna add in analytics js next